src/ISA/ISA.Json/Assay.fs

```diff
@@ -71,10 +71,10 @@ module Assay =
             }
         )
 
-    let fromString (s:string) = 
-        GDecode.fromString (decoder (ConverterOptions())) s
+    let decodeFromString (s:string) = 
```

Comment:
@kMutagene
What do you think about this function naming?

The function has a json-string as input and returns the decoded object.

Reply:
is "decode" a toth thing? nonetheless, people usually expect this to be called deserialize